javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,9 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  var suffixArray = [];
+  for (var i = 0; i < words.length; i++) {
+    words[i] += suffix;
```

Comment:
Oh nooooo, this is changing the original array. Please try to do this without changing the values stored in the argument array.